fiddler/metadata.csv

```diff
@@ -16,8 +16,11 @@ fiddler.mae,gauge,,percent,,The MAE of the model,0,fiddler,mae,
 fiddler.mse,gauge,,percent,,The MSE of the model,0,fiddler,mse,
 fiddler.r2,gauge,,percent,,The R-squared metric of the model,0,fiddler,r2,
 fiddler.binary_cross_entropy,gauge,,percent,,The binary cross-entropy metric of the model,0,fiddler,binary_cross_entropy,
-fiddler.expected_callibration_error,gauge,,percent,,The expected calibration error of the model,0,fiddler,expected_callibration_error,
```

Comment:
Shouldn't we consider this a breaking change if we're renaming metrics? Imagine someone who has a dashboard or monitor based on this metric? It's fine to make the change, but then we should bump the version to 2.0.0.

An alternative (which lets us keep 1.1.0) is to keep submitting metrics with the old wrong names **in addition to the correct names** and add a note to their descriptions saying they are deprecated give the names of the metrics with correct names as equivalents.

Reply:
Gotcha! Good to know that metric name updates require major version changes.
Have updated the version to be 2.0.0.
Thanks for the review! 🙏